style.css

```diff
@@ -509,3 +522,304 @@ textarea::placeholder {
+
+  .cardImg2 {
+    content: url(assets/bigSizePicCard2.svg);
+  }
+
+  /* card3 img */
+
+  .cardImg3 {
+    content: url(assets/bigSizePicCard3.svg);
+  }
+
+  /* card4 img */
+  .cardImg4 {
+    content: url(assets/bigSizePicCard4.svg);
+  }
+
+  /* Invert img */
+  .card-container:nth-child(4) .cardImg,
+  .card-container:nth-child(6) .cardImg {
+    order: 1;
+  }
+
+  /* Word TONIC */
+  .cardWorks h4 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    margin-left: 2.2rem;
+  }
+
+  /* Class card work skills setting */
+  .cardWorks-skills {
+    margin-top: 1rem;
+  }
+
+  /* Words "canopy" "back end dev" and "2015" settings */
+  .word1 {
+    width: 43px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .word2 {
+    width: 120px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    padding: 8px;
+    color: #6b778c;
+  }
+
+  .word3 {
+    padding: 8px;
+    width: 38px;
+    height: 24px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  /* settings for pharagarph */
+  .cardWorks .pCards {
+    margin-top: 1rem;
+    width: 31rem;
+    height: 50px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  /* settings for words: html css and javascript */
+  .workContainer {
+    margin-top: 1rem;
+    border-radius: 8px;
+    margin-left: 2.2rem;
+  }
+
+  /* settings for words: html css and javascript */
+  .cardButton {
+    width: 140px;
+    margin-left: 2.2rem;
+    margin-top: 3rem;
+  }
+
+  /* About myself */
+  .aboutMyself {
+    display: flex;
+    flex-direction: row;
+    background-color: white;
+    border-top-right-radius: 6rem;
+    gap: 3rem;
+  }
+
+  .aboutMyself-container {
+    width: 40%;
+    margin-left: 3rem;
+  }
+
+  .aboutMyself-container h3 {
+    margin-top: 2rem;
+  }
+
+  .desktopVersion {
+    width: 45%;
+  }
+
+  .aboutMyself-container p {
+    padding: 0;
+    margin: 0;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 30px;
+  }
+
+  .span-language {
+    padding: 0;
+    margin: 0;
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 35px;
+    color: #344563;
+  }
+
+  .languagesContainer {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .container-desktop {
+    display: flex;
+    flex-direction: row;
+    gap: 3px;
+  }
+
+  .languages {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    background-color: rgba(247, 247, 249, 1);
+    width: 10rem;
+    height: 8rem;
+  }
+
+  #js {
+    padding: 0;
+    padding-top: 0.5rem;
+    width: 6rem;
+  }
+
+  #html {
+    padding: 0;
+    padding-top: 0.5rem;
+    width: 3rem;
+  }
+
+  #css {
+    padding: 0;
+    padding-top: 0.5rem;
+    width: 2rem;
+  }
+}
```

Comment:
You seem to not be having a proper view of the background images in the contact section, could you please add all the images in the section and also remember to place them in the right position similar to what we have in our figma template 😄 

You should also implement the following changes for your contact section 👍 

1. The text after the section title is aligned in three lines as seen in the Figma template, could you please implement this in your project
2. Please I think you should increase the width of your contact-form input to be similar to the design and also use the right font styling given in our design
3. kindly align the `Get in touch` button to the center and use the font-styling given in the template 

Figma template :
![image](https://github.com/fmalgarini/My-portfolio-Project/assets/20537712/7602ec4a-1e04-4bea-a45c-43bb2b70a592)

Your design :
![image](https://github.com/fmalgarini/My-portfolio-Project/assets/20537712/99036895-c92a-418d-9a6b-03b5001e3a10)